development/shellcheck.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# lint all *.sh files
+shellcheck --version && find . -type f -name '*.sh' ! -path './node_modules/*' -print0 | xargs -0 shellcheck
```

Comment:
This script doesn't seem to fail when `shellcheck` fails.

Reply:
I'd suggest using a few Bash flags to make this script safer by default, such as `-e`, `-u`, and `-o pipefail`. Maybe `-x` as well so that it's easier to see what's happening on CI.